mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -269,11 +305,13 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD ac setOnFireFromLava ()V
 	METHOD ad extinguish ()V
 	METHOD ae destroy ()V
+	METHOD af calculateNextStepDistance ()F
 	METHOD ag moveToBoundingBoxCenter ()V
 	METHOD ah getSwimSound ()Lyk;
 	METHOD ai getSplashSound ()Lyk;
 	METHOD aj getHighSpeedSplashSound ()Lyk;
 	METHOD ak checkBlockCollision ()V
+	METHOD al isJesus ()Z
```

Comment:
needs changing, I have no clue what this is supposed to mean.

Reply:
Is `walksOnAir` better?

That method controls whether an entity plays stepping sounds when in the air, so probably overriden by bats. I don't know what the correct term for it would be.

Reply:
> probably overridden by bats

Have you not look at implementations? *Always* look at implementations and uses when mapping a member.

Reply:
I look at the uses. I'm used to not being able to easily find implementations. (iirc the only thing I've used that had that feature was Android Studio)

I did later check and eventually found the correct place where it's being used is in `ParrotEntity` (overriden to return true)

Reply:
When's it used? When it's on the player's shoulder or what?

Reply:
It's used in `move()` on the base entity class and just as a check whether to update stepping sounds when not on the ground. (there are separate timers for that).

imo It's used when the Parrots fly so they can make wing flapping sounds.